LayoutTests/inspector/unit-tests/set-utilities.html

```diff
@@ -17,6 +17,24 @@
         },
     });
 
+    suite.addTestCase({
+        name: "Set.prototype.filter",
+        test() {
+            function test(set, callback, thisArg, expected, message) {
+                InspectorTest.expectShallowEqual(Array.from(set.filter(callback, thisArg)), expected, message);
+            }
+
+            let set1 = new Set([1, 2, 3, 4]);
+            let set2 = new Set([2, 4]);
```

Comment:
Nit: These sets aren't mutated since `filter` is not in-place, so it might be clearer to make them `const`.